operations/environments/dev/main.tf

```diff
@@ -20,7 +20,6 @@ provider "azurerm" {
   features {
     key_vault {
       purge_soft_deleted_secrets_on_destroy = false
-      purge_soft_deleted_keys_on_destroy    = false
```

Comment:
Looks like this code has a lot of changes that were reverted from `main`, including this line.  And it looks like it was reverted in such a way that the code would be undone if this PR is to be merged.